From mboxrd@z Thu Jan  1 00:00:00 1970
From: Venkatesh Odela <venkatesh.odela@amd.com>
Subject: [LINUX PATCH] riscv: configs: amd: Enable DP83867 PHY for SCU200MBV
Date: Tue, 28 Jul 2026 19:37:19 +0530
Message-ID: <20260728140719.1462882-1-venkatesh.odela@amd.com>
Content-Transfer-Encoding: 8bit
Return-Path: <>
MIME-Version: 1.0
To: git-dev@amd.com
Cc: harini.katakam@amd.com
List-ID: <git-dev@amd.com>
Content-Type: text/plain; charset="utf-8"

SCU200MBV uses a TI DP83867 Ethernet PHY with the Xilinx AXI EMAC.
Enable CONFIG_DP83867_PHY so the PHY is detected and configured at
boot.

Signed-off-by: Venkatesh Odela <venkatesh.odela@amd.com>

Upstream-Status: Backport from newer kernel

---
 arch/riscv/configs/amd.config | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/riscv/configs/amd.config b/arch/riscv/configs/amd.config
index dc1788e2d22d..19b4d2689fa0 100644
--- a/arch/riscv/configs/amd.config
+++ b/arch/riscv/configs/amd.config
@@ -5,6 +5,7 @@
 
 CONFIG_XILINX_AXI_EMAC=y
 CONFIG_MARVELL_PHY=y
+CONFIG_DP83867_PHY=y
 CONFIG_SERIAL_UARTLITE=y
 CONFIG_SERIAL_UARTLITE_CONSOLE=y
 CONFIG_GPIO_XILINX=y
-- 
2.25.1



